src/app/hooks/useToggle/README.md

```diff
@@ -0,0 +1,20 @@
+# useToggle Hook
```

Comment:
We'd need to update this readme once we move all toggles to the remote features toggles as at they point toggles become service specific and not global anymore.